src/spinner/spinner.style.ts

```diff
@@ -17,14 +18,18 @@ export default css`
     animation: rotate 1s linear infinite;
   }
 
-  .primary {
+  .normal {
     color: var(--tap-spinner-color-primary, var(--tap-sys-color-surface-black));
   }
 
   .inverse {
     color: var(--tap-spinner-color-inverse, var(--tap-sys-color-surface-white));
   }
 
+  .inherit {
+    color: inherit;
```

Comment:
The color property is an inherited property. the `{ color: inherit; }` does nothing.